ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseProxy.java

```diff
@@ -43,24 +82,30 @@ public PubsubMessage apply(PubsubMessage message) {
       // Prevent null pointer exception
       message = PubsubConstraints.ensureNonNull(message);
 
+      if (proxyMatcher == null) {
+        loadResourcesOnFirstMessage();
```

Comment:
I think it's generally better to do initializations in `@Setup` like in https://github.com/mozilla/gcp-ingestion/pull/2224/files#diff-6135a4da74da3d17c14be1739ed0c54b9643750f2ef0df5575cd3ca1bca74edfR99. Although I don't expect this to cause any trouble due to size of the lists.